src/tests/test_connection.py

```diff
@@ -717,12 +717,18 @@ def test_feedback_when_switching_connection_with_existing_connection(
     assert "Switching to connection one" == captured.out.splitlines()[-1]
 
 
-def test_no_feedback(ip_empty, tmp_empty, capsys):
+@pytest.mark.parametrize(
```

Comment:
this needs a bit more testing, can you check that the message is shown both when feedback is 1,2 and that it isn't shown when feedback is 0?

Reply:
There was testing feedbacks when feedback=0, so I changed the name of the function: [fixed](https://github.com/ploomber/jupysql/pull/834/files#diff-ecfdf2a1f3adc925d3c32b8aa52540474a10a4160d552f281d7dbd85a0260170R740-R749)

For feedback=1 and 2, [fixed](https://github.com/ploomber/jupysql/pull/834/files#diff-ecfdf2a1f3adc925d3c32b8aa52540474a10a4160d552f281d7dbd85a0260170R723-R726)